examples/ConfigurableFirmataWiFi/ConfigurableFirmataWiFi.ino

```diff
@@ -317,6 +343,9 @@ char wep_key[] = "your_wep_key";
 #elif defined(ESP8266_WIFI) && defined(SERIAL_DEBUG)
 #define IS_IGNORE_PIN(p)  ((p) == 1)
 
+#elif defined(ESP32_WIFI) && defined(SERIAL_DEBUG)
+#define IS_IGNORE_PIN(p)  ((p) == 1 || (p) == 3)
```

Comment:
I have not been able to determine if this is absolutely necessary for the ESP32 since the connection is so flaky.